Checkpoint.html

```diff
@@ -17,7 +17,8 @@
    </nav>
 
 <body>
-  <h1><b>Welcome to Engineer's Bahaa Website</b></h1>
+
+  <h1><b>WELCOME to Engineer's Bahaa Website</b></h1>
```

Comment:
No need for `<b>` here, HTML's `<h1>` already come with default styling of extrabold. Also, let's try to keep styling concerns to CSS.